editor/editor_file_dialog.cpp

```diff
@@ -558,7 +558,9 @@ void EditorFileDialog::_item_list_item_rmb_selected(int p_item, const Vector2 &p
 	}
 	if (single_item_selected) {
 		item_menu->add_separator();
-		item_menu->add_icon_item(get_icon("Filesystem", "EditorIcons"), TTR("Show In File Manager"), ITEM_MENU_SHOW_IN_EXPLORER);
+		Dictionary item_meta = item_list->get_item_metadata(p_item);
+		String item_text = (item_meta["dir"]) ? TTR("Open In File Manager") : TTR("Show In File Manager");
```

Comment:
Style-wise, I wouldn't put parentheses around `item_meta["dir"]` (only if it was an expression with an evaluation like `(item_meta["dir"] != other_thing)`). Same for the other ternary operator in `filesystem_dock.cpp`.

Reply:
Will change, but there are other cases not written by me following that style, even on the same file. It seems to be mixed but I'd leave normalizing that for a different patch.